frontendActions/src/main/java/com/virtuslab/gitmachete/frontend/actions/BaseRebaseBranchOntoParentAction.java

```diff
@@ -62,15 +62,14 @@ public void update(AnActionEvent anActionEvent) {
     if (state != Repository.State.NORMAL) {
 
       var stateName = Match(state).of(
-          Case($(Repository.State.GRAFTING), "during an ongoing cherry-pick"),
-          Case($(Repository.State.DETACHED), "in the detached head state"),
-          Case($(Repository.State.MERGING), "during an ongoing merge"),
-          Case($(Repository.State.REBASING), "during an ongoing rebase"),
-          Case($(Repository.State.REVERTING), "during an ongoing revert"),
+          Case($(Repository.State.GRAFTING), "ongoing cherry-pick or revert"),
+          Case($(Repository.State.DETACHED), "detached head state"),
+          Case($(Repository.State.MERGING), "ongoing merge"),
+          Case($(Repository.State.REBASING), "ongoing rebase"),
```

Comment:
Pls drop a comment what happens here in 2019.3+ and that we deliberately skip the `REVERTING` case

Reply:
Or you could also sth like `var reverting = Try.of(() -> Repository.State.valueOf("REVERTING")).getOrNull()` and match `state` against that; you'd also need sth like:

```java
Case($(Repository.State.GRAFTING), reverting != null ? "ongoing cherry-pick" : "ongoing cherry-pick or revert"),
Case($(reverting), "ongoing cherry-pick" : "ongoing revert"),
```